server/server.js

```diff
@@ -24,6 +25,10 @@ if (process.env.NODE_ENV === 'development') {
   enableDevMiddleWare(app);
 }
 
+if (process.env.NDLA_ENVIRONMENT === 'test') {
```

Comment:
Nice, men funka det ikke å "slå av" basic auth for knowit ip adressen også?

Reply:
Hadde glemt det, men tror jeg har noe som funker nå.